test/e2e/tests/configmap_update/eraser_test.go

```diff
@@ -22,7 +22,7 @@ import (
 
 const (
 	numPods        = 3
-	collectorLabel = "name=remover"
+	collectorLabel = "eraser.sh/type=manual"
```

Comment:
Move the constant into the test utils package since it's used in many different tests.